docs/guides/javascript.md

```diff
@@ -23,6 +23,7 @@ spectral
 ```
 
 Please note that by default Spectral supports YAML 1.2 with merge keys extension.
+As recommended by the [spec](https://yaml.org/spec/1.2/spec.html#id2804923), tags are resolved according to the "Core Schema".
```

Comment:
On a slightly unrelated note - we could move these 2 sentences elsewhere in future, to some more general document.